commands/eval.js

```diff
@@ -1,5 +1,5 @@
 const config = require("../config.json");
-const core = require("../coreFunctions.js");
```

Comment:
`coreFunctions` is included in order for easily evaling with `core.whatever()` without having to first require core.

Reply:
Why not remove it now, and just add it when you need it?